src/keylessBackup/KeylessBackupPhoneCodeInput.tsx

```diff
@@ -21,15 +22,53 @@ import { Spacing } from 'src/styles/styles'
 import VerificationCodeInput from 'src/verify/VerificationCodeInput'
 
 function HelpInfoBottomSheet({
-  onPressHelpGoBack,
```

Comment:
This was changed from the function being passed to values used within the component to create the appropriate functions. If we passed the functions we would need to conditionally pass different functions, is that correct? If so this seems fine.

Reply:
Just moved closer to where the functions are really used. We can still pass the 3 functions and conditionally decide what function to use as now, but the parent component doesn't need to define that.